### PR TITLE
Add magdeburger developer days

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Add event,
Add logo,
Add Marco's talk.

Screenshot 2023-03-13 at 11 17 27
929de7e36aa.png">

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
Hi @marcoow. I have added the event you tagged me to the calendar. I will add it to the appearances section when/if a video is released. I am in touch with David regarding the logo to make it fit. Also, I wrote the description in German using your talk's description, hope it works.

**Review by marcoow — CHANGES_REQUESTED** [review]
all of our website should remain in english

**Comment by emmasofiah2o** [comment]
@marcoow updated

File: src/calendar/2023-05-08-magdeburger-developer-days.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Magdeburger Developer Days"
+image: "/assets/images/calendar/2023-05-08-magdeburger-developer-days/mdd-logo-background.png"
+location: "Magdeburg"
+url: https://md-devdays.de/home
+kind: conference
+color: aqua
+---
+
+Magdeburger Developer Days is a 3-day developer & community conference in
```
Comment (marcoow) [inline src/calendar/2023-05-08-magdeburger-developer-days.md]:
```suggestion
Magdeburger Developer Days is a 3-day software engineering conference in
```

File: src/calendar/2023-05-08-magdeburger-developer-days.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Magdeburg. This year, Mainmatter Founder Marco Otte-Witte will talk about
+Continuous Deployment Workflows. In his talk, he will present main advantages
+compared to classical processes, explain the essential components of such a
+process (especially with ryegard to testing and automation), and discuss typical
+challenges and strategies to overcome them.
```
Comment (marcoow) [inline src/calendar/2023-05-08-magdeburger-developer-days.md]:
```suggestion
Magdeburg. Mainmatter Founder Marco Otte-Witte will talk about
Continuous Deployment Workflows. He will present main advantages
compared to classical processes, explain the essential components of such a
process (especially with ryegard to testing and automation), and discuss typical
challenges and strategies to overcome them.
```

File: src/calendar/2023-05-08-magdeburger-developer-days.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+url: https://md-devdays.de/home
+kind: conference
+color: aqua
+---
+
+Magdeburger Developer Days is a 3-day software engineering conference in
+Magdeburg. Mainmatter Founder Marco Otte-Witte will talk about Continuous
+Deployment Workflows. He will present main advantages compared to classical
+processes, explain the essential components of such a process (especially with
+ryegard to testing and automation), and discuss typical challenges and
```
Comment (emmasofiah2o) [inline src/calendar/2023-05-08-magdeburger-developer-days.md]:
```suggestion
regard to testing and automation), and discuss typical challenges and
```

**Comment by marcoow** [comment]
@emmasofiah2o linting fails

**Comment by emmasofiah2o** [comment]
Can I merge @marcoow?

**Review by marcoow — APPROVED** [review]

**Pull request merged**